guides/Rakefile

```diff
@@ -10,16 +10,19 @@ namespace :guides do
       ruby "-Eutf-8:utf-8", "rails_guides.rb"
     end
 
-    desc "Generate .mobi file. The kindlegen executable must be in your PATH. You can get it for free from http://www.amazon.com/gp/feature.html?docId=1000765211"
+    desc "Generate .mobi file"
     task :kindle do
-      require "kindlerb"
-      unless Kindlerb.kindlegen_available?
-        abort "Please run `setupkindlerb` to install kindlegen"
-      end
+      require "active_support/deprecation"
+      ActiveSupport::Deprecation.warn("The guides:generate:kindle rake task is deprecated and will be removed in 7.2. Run rake guides:generate:epub instead")
+      Rake::Task["guides:generate:epub"].invoke
+    end
+
+    desc "Generate .epub file"
+    task :epub do
       unless /convert/.match?(`convert`)
         abort "Please install ImageMagick"
       end
```

Comment:
When I run the task I get:

```
DEPRECATION WARNING: The guides:generate:kindle rake task is deprecated and will be removed in 7.2. Run rake guides:generate:epub instead (called from block in execute at /Users/eileencodes/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/rake-13.0.6/lib/rake/task.rb:281)
rake aborted!
Errno::ENOENT: No such file or directory - convert
/Users/eileencodes/open_source/real_rails/guides/Rakefile:22:in ``'
/Users/eileencodes/open_source/real_rails/guides/Rakefile:22:in `block (3 levels) in <top (required)>'
/Users/eileencodes/open_source/real_rails/guides/Rakefile:17:in `block (3 levels) in <top (required)>'
/Users/eileencodes/.rbenv/versions/3.1.1/bin/bundle:25:in `load'
/Users/eileencodes/.rbenv/versions/3.1.1/bin/bundle:25:in `<main>'
```

I know you didn't write this section but I can't test previous behavior because kindlegen wont install. Is it possible we no longer need this now that we're not using kindlegen? Does this work locally for you? I'm on an M1 mac

Reply:
@eileencodes Mine seem to run smoothly. I think we do not need this anymore

Reply:
You are right. We no longer require this. I have removed this line now.